### PR TITLE
Sprint 50: TLT-2490 - people tool jasmine tests

### PR DESCRIPTION
**Pull request by elliottyates**

### CODE REVIEW

File: people_tool/tests/jasmine/SearchControllerTests.js [hunk 1]
```diff
@@ -51,15 +47,135 @@ describe('Unit testing people_tool SearchController', function() {
...
+
+        it('does not hit backend if search string is too long', function() {
+            // at controller initialization, default search term has a maximum
+            // length of 32 characters
+            scope.queryString = '123456789012345678901234567890123';
+            scope.searchPeople();
+            expect($timeout.flush).toThrowError('No deferred tasks to be flushed');
+            expect(scope.dtInstance.reloadData).not.toHaveBeenCalled();
+            expect(scope.messages.length).toBe(1);
+            expect(scope.messages[0].type).toEqual('warning');
```
Comment (sapnamysore) [inline people_tool/tests/jasmine/SearchControllerTests.js]: Could the searchType(eg. univ_id) have been set additionally and the specific error text have been validated? I see the advantage of keeping it generic, but we could target specific cases with the searchType.

File: people_tool/tests/jasmine/SearchControllerTests.js [hunk 2]
```diff
@@ -51,15 +47,135 @@ describe('Unit testing people_tool SearchController', function() {
...
+            digestSpy = spyOn(scope, '$digest');
+        });
+
+        afterEach(function() {
+            expect(toggleOperationInProgressSpy).toHaveBeenCalledWith(false);
+            expect(digestSpy).toHaveBeenCalledTimes(1);
+        });
+
+        it('sends expected GET params to backend', function() {
+            ajaxSpy.and.callFake(respondWithData);
```
Comment (sapnamysore) [inline people_tool/tests/jasmine/SearchControllerTests.js]: Nice. Haven't used <f1> .and.<f2>

**Comment by sapnamysore** [comment]
👍

File: people_tool/tests/jasmine/SearchControllerTests.js (hunk 1 above)
Comment (elliottyates) [inline people_tool/tests/jasmine/SearchControllerTests.js]: Ok - I can look into striking a balance between generic and specific by checking each of the `searchType`s.

**Pull request merged**